addons/transform/org.openhab.transform.feed/README.MD

```diff
@@ -0,0 +1,9 @@
+# Feed Transformation Service
+
+Feed transformation service transforms a feed between some of the supported feed formats: RSS 0.9, RSS 0.91 Netscape, RSS 0.91 Userland, RSS 0.92, RSS 0.93, RSS 0.94, RSS 1.0, RSS 2.0, Atom 0.3, Atom 1.0.
+
+
+
+## Example
+
+Examples can be found in the test bundle.
```

Comment:
This read me is also for docs.openhab.org and I think we should refer our end users to the test cases in the code.